pkg/cmd/backuprestore/backupserver.go

```diff
@@ -48,10 +55,32 @@ func (b *backupServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&b.schedule, "schedule", "", "schedule specifies the cron schedule to run the backup")
 	fs.StringVar(&b.timeZone, "timezone", "", "timezone specifies the timezone of the cron schedule to run the backup")
 
+	cobra.MarkFlagRequired(fs, "schedule")
```

Comment:
@tjungblu we cant make this required, because if the container is disabled, which is the case initially, the staticPod will fail to launch 

[prow](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_cluster-etcd-operator/1306/pull-ci-openshift-cluster-etcd-operator-master-e2e-operator/1831311344830255104/artifacts/e2e-operator/gather-extra/artifacts/pods/openshift-etcd_etcd-ip-10-0-37-122.ec2.internal_etcd-backup-server.log)

Reply:
makes sense